### PR TITLE
Fix emails + other template fixes

### PR DESCRIPTION
**Pull request by carakas**
Changelog:

- SITE_URL and SITE_DOMAIN were only available in backend templates, this has been fixed since they are used in emails
- On numerous places form field names where cased wrong in the frontend (triton)
- The backend can now use templates from the frontend to mail
- Fix typo in mass action forms action that caused the form to be submitted to the wrong page

### CODE REVIEW

File: src/Backend/Modules/FormBuilder/Layout/Templates/DataDetails.html.twig [hunk 1]
```diff
@@ -48,7 +48,7 @@
           <table class="table">
             <tr>
               <th>{{ 'lbl.SentOn'|trans|capitalize }}:</th>
-              <td>{{ sentOn|formatdatetime }}</td>
+              <td>{{ sentOn|spoondate() }}</td>
```
Comment (WouterSioen) [inline src/Backend/Modules/FormBuilder/Layout/Templates/DataDetails.html.twig]: are the brackets needed when you don't send any parameters?

File: src/Backend/Modules/ContentBlocks/Installer/Data/install.sql [hunk 2]
```diff
@@ -3,7 +3,7 @@ CREATE TABLE IF NOT EXISTS `content_blocks` (
  `revision_id` int(11) NOT NULL auto_increment,
  `user_id` int(11) NOT NULL,
  `extra_id` int(11) NOT NULL,
- `template` varchar(255) NOT NULL default 'default.tpl',
+ `template` varchar(255) NOT NULL default 'Default.html.twig',
```
Comment (WouterSioen) [inline src/Backend/Modules/ContentBlocks/Installer/Data/install.sql]: this has to be changed in the database for the tests too (tools/xxx.sql)

File: src/Frontend/Modules/Faq/Layout/Templates/Mails/Feedback.html.twig [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% extends 'Core/Layout/Templates/Mails/Default.html.twig' %}
+
+{% block main %}
+  <h2>{{ SITE_TITLE }}: {{ 'msg.FaqFeedbackSubject'|trans|format( question ) }}</h2>
```
Comment (jeroendesloovere) [inline src/Frontend/Modules/Faq/Layout/Templates/Mails/Feedback.html.twig]: Can you remove spaces before and after "question"? `|trans|format(question)`

File: src/Frontend/Modules/Faq/Layout/Templates/Mails/OwnQuestion.html.twig [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{% extends 'Core/Layout/Templates/Mails/Default.html.twig' %}
+
+{% block main %}
+  <h2>{{ SITE_TITLE }}: {{ 'msg.FaqOwnQuestionSubject'|trans|format( name ) }}</h2>
```
Comment (jeroendesloovere) [inline src/Frontend/Modules/Faq/Layout/Templates/Mails/OwnQuestion.html.twig]: Can you remove space before/after name?

File: src/Frontend/Modules/FormBuilder/Layout/Templates/Mails/Form.html.twig [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{% extends 'Core/Layout/Templates/Mails/Default.html.twig' %}
+
+{% block main %}
+  <h2>{{ SITE_TITLE }}: {{ 'msg.FormBuilderSubject'|trans|format( name ) }}</h2>
```
Comment (jeroendesloovere) [inline src/Frontend/Modules/FormBuilder/Layout/Templates/Mails/Form.html.twig]: Idem

File: src/Backend/Modules/ContentBlocks/Installer/Data/install.sql (hunk 2 above)
Comment (carakas) [inline src/Backend/Modules/ContentBlocks/Installer/Data/install.sql]: it has been https://github.com/forkcms/forkcms/pull/1566/files#diff-32212ef9f7bc7b6d110619bd2798369bR174

File: src/Frontend/Modules/Profiles/Layout/Templates/Mails/ForgotPassword.html.twig [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{% extends 'Core/Layout/Templates/Mails/Default.html.twig' %}
+
+{% block main %}
+  <h2>{{ 'msg.ForgotPasswordSubject'|trans }}</h2>
+  <hr /><br />
+  <p>{{ 'msg.ForgotPasswordSalutation'|trans }}</p>
+  <p>{{ 'msg.ForgotPasswordBody'|trans|format( SITE_URL, resetUrl)|raw }}</p>
```
Comment (jeroendesloovere) [inline src/Frontend/Modules/Profiles/Layout/Templates/Mails/ForgotPassword.html.twig]: Can you remove space before SITE_URL?

File: src/Frontend/Modules/Profiles/Layout/Widgets/LoginBox.html.twig [hunk 7]
```diff
@@ -8,7 +8,7 @@
   <div class="inner">
     {% if isLoggedIn %}
       <div class="bd content">
-        {{ 'msg.ProfilesLoggedInAs'|trans|sprintf({$profile.display_name }}:{{ profile.url.dashboard }}}) -
+        {{ 'msg.ProfilesLoggedInAs'|trans|format( profile.display_name, profile.url.dashboard )|raw }} -
```
Comment (jeroendesloovere) [inline src/Frontend/Modules/Profiles/Layout/Widgets/LoginBox.html.twig]: Can you remove space before/after variables in format() please.

File: src/Frontend/Modules/Profiles/Layout/Widgets/LoginLink.html.twig [hunk 8]
```diff
@@ -5,7 +5,7 @@
 #}
 
 {% if isLoggedIn %}
-  {{ 'msg.ProfilesLoggedInAs'|trans|sprintf({$profile.display_name }}:{{ profile.url.dashboard } }})
+  {{ 'msg.ProfilesLoggedInAs'|trans|format( profile.display_name, profile.url.dashboard )|raw }}
```
Comment (jeroendesloovere) [inline src/Frontend/Modules/Profiles/Layout/Widgets/LoginLink.html.twig]: idem

File: src/Backend/Modules/ContentBlocks/Installer/Data/install.sql (hunk 2 above)
Comment (WouterSioen) [inline src/Backend/Modules/ContentBlocks/Installer/Data/install.sql]: oops, looked over it. thx!

File: src/Frontend/Modules/Profiles/Layout/Widgets/LoginLink.html.twig (hunk 8 above)
Comment (carakas) [inline src/Frontend/Modules/Profiles/Layout/Widgets/LoginLink.html.twig]: fixed in 5942bf8

File: src/Frontend/Modules/Profiles/Layout/Widgets/LoginBox.html.twig (hunk 7 above)
Comment (carakas) [inline src/Frontend/Modules/Profiles/Layout/Widgets/LoginBox.html.twig]: fixed in 5942bf8

File: src/Frontend/Modules/Profiles/Layout/Templates/Mails/ForgotPassword.html.twig (hunk 6 above)
Comment (carakas) [inline src/Frontend/Modules/Profiles/Layout/Templates/Mails/ForgotPassword.html.twig]: fixed in 5942bf8

File: src/Frontend/Modules/FormBuilder/Layout/Templates/Mails/Form.html.twig (hunk 5 above)
Comment (carakas) [inline src/Frontend/Modules/FormBuilder/Layout/Templates/Mails/Form.html.twig]: fixed in 5942bf8

File: src/Frontend/Modules/Faq/Layout/Templates/Mails/OwnQuestion.html.twig (hunk 4 above)
Comment (carakas) [inline src/Frontend/Modules/Faq/Layout/Templates/Mails/OwnQuestion.html.twig]: fixed in 5942bf8

File: src/Frontend/Modules/Faq/Layout/Templates/Mails/Feedback.html.twig (hunk 3 above)
Comment (carakas) [inline src/Frontend/Modules/Faq/Layout/Templates/Mails/Feedback.html.twig]: fixed in 5942bf8

File: src/Backend/Modules/FormBuilder/Layout/Templates/Data.html.twig [hunk 9]
```diff
@@ -51,28 +51,29 @@
...
-                <p>{{ 'msg.ConfirmMassDelete'|trans|capitalize }}</p>
-              </div>
-              <div class="modal-footer">
-                <button type="button" class="btn btn-default" data-dismiss="modal">
-                  <span class="fa fa-times"></span>{{ 'lbl.Cancel'|trans|capitalize }}
-                </button>
-                <button type="submit" class="btn btn-primary">
-                  <span class="fa fa-trash"></span>{{ 'lbl.Delete'|trans|capitalize }}
-                </button>
+        <form action="{{ geturl('mass_data_action' ) }}" method="get" class="forkForms">
```
Comment (carakas) [inline src/Backend/Modules/FormBuilder/Layout/Templates/Data.html.twig]: fixed in 5942bf8

**Pull request merged**